interface-definitions/load-balancing-haproxy.xml.in

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="load-balancing">
+    <children>
+      <node name="reverse-proxy" owner="${vyos_conf_scripts_dir}/load-balancing-haproxy.py">
+        <properties>
+          <help>Configure reverse-proxy load-balancing</help>
+        </properties>
+        <children>
+          <tagNode name="server">
+            <properties>
+              <help>Frontend name</help>
+            </properties>
+            <children>
+              <leafNode name="backend">
+                <properties>
+                  <help>Backend member</help>
```

Comment:
Should we limit backend server names to be alphanumeric only? It's a "virtual" identifier inside haproxy conf. I guess we do not wan't to have fancy characters like äöüß

Reply:
Done!